modules/standard/Path.chpl

```diff
@@ -592,36 +627,51 @@ proc normPath(name: string): string {
   return result;
 }
 
-/* Given a path ``name``, attempts to determine the canonical path referenced.
+pragma "no doc"
+pragma "last resort"
+proc normPath(name: string): string {
+  compilerWarning('Argument `name` is deprecated - use `path` instead');
+  return normPath(path=name);
+}
+
+/* Given a path ``path``, attempts to determine the canonical path referenced.
    This resolves and removes any :data:`curDir` and :data:`parentDir` uses
    present, as well as any symbolic links.  Returns the result.
 
-   :arg name: A path to resolve.  If the path does not refer to a valid file
+   :arg path: A path to resolve.  If the path does not refer to a valid file
               or directory, an error will occur.
-   :type name: `string`
+   :type path: `string`
 
    :return: A canonical version of the argument.
    :rtype: `string`
    :throws SystemError: If one occurs.
 */
-proc realPath(name: string): string throws {
+proc realPath(path: string): string throws {
   extern proc chpl_fs_realpath(path: c_string, ref shortened: c_string): syserr;
 
   var res: c_string;
-  var err = chpl_fs_realpath(unescape(name).c_str(), res);
-  if err then try ioerror(err, "realPath", name);
+  var err = chpl_fs_realpath(unescape(path).c_str(), res);
+  if err then try ioerror(err, "realPath", path);
   const ret = createStringWithNewBuffer(res, policy=decodePolicy.escape);
   // res was qio_malloc'd by chpl_fs_realpath, so free it here
   chpl_free_c_string(res);
   return ret; 
 }
 
 pragma "no doc"
-proc realPath(out error: syserr, name: string): string {
+pragma "last resort"
+proc realPath(name: string): string throws {
+  compilerWarning('Argument `name` is deprecated - use `path` instead');
+  return realPath(path=name);
+}
+
+
+pragma "no doc"
+proc realPath(out error: syserr, path: string): string {
```

Comment:
I realize this is not part of this PR but I would just drop these `out error` versions altogether. I believe they are very old.

Note that there might be some tests that are still using them. I'd run paratest *after* removing these.

Reply:
If you have already run some testing, you can leave them in for now.